src/chrome/content/rules/Axios.com.xml

```diff
@@ -0,0 +1,17 @@
+<!--
+	Nonfunctional hosts in *.axios.com:
+
+	h: http redirect
+	m: certificate mismatch
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
```

Comment:
These comments aren't really required if there are no nonfunctional subdomains. This applies to all other rulesets.

Reply:
This boilerplate's generated by the trivial rule script and I left it in to be filled out later as necessary. Better to remove?

Reply:
For me at least it's better to have them removed :)